packages/core/src/modules/mdoc/MdocService.ts

```diff
@@ -37,6 +40,29 @@ export class MdocService {
     return await mdoc.verify(agentContext, options)
   }
 
+  public async getKeyFromMdocCredentialHolderBinding(
+    agentContext: AgentContext,
+    holderBinding: MdocCredentialHolderBinding
+  ) {
+    let holderKey: Key
+    if (holderBinding.method !== 'jwk') {
```

Comment:
```suggestion
    if (holderBinding.method === 'did') {
```


But again, i don't think we should allow the did holder binding without the did actually being put in the mdoc (like is the case with sd-jwt, the binding actually represents what gets put in the cred, did or jwk). So do we need mdoc holder binding at all? I think jwk is closest to what gets put in the actual mdoc